airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/state_converters/abstract_stream_state_converter.py

```diff
@@ -19,13 +19,13 @@ class AbstractStreamStateConverter(ABC):
     END_KEY = "end"
 
     def get_concurrent_stream_state(
-        self, cursor_field: Optional["CursorField"], state: MutableMapping[str, Any]
+        self, cursor_field: Optional["CursorField"], start: Any, state: MutableMapping[str, Any]
     ) -> Optional[MutableMapping[str, Any]]:
         if not cursor_field:
             return None
         if self.is_state_message_compatible(state):
-            return self.deserialize(state)
-        return self.convert_from_sequential_state(cursor_field, state)
+            compatible_state = self.deserialize(state)
```

Comment:
what is the `compatible_state` variable used for?

Reply:
`compatible_state` is no longer used.